src/query/storage/converter_test.go

```diff
@@ -307,23 +307,30 @@ func TestPromTimeSeriesToSeriesAttributesPromMetricsType(t *testing.T) {
 	}
 
 	mapping := map[prompbMetricTypeWithNameSuffix]promMetricTypeWithBool{
-		{metricType: prompb.MetricType_UNKNOWN}:  {metricType: ts.PromMetricTypeUnknown},
-		{metricType: prompb.MetricType_COUNTER}:  {metricType: ts.PromMetricTypeCounter, handleValueResets: true},
+		{metricType: prompb.MetricType_UNKNOWN}: {metricType: ts.PromMetricTypeUnknown},
+
+		{metricType: prompb.MetricType_COUNTER}: {metricType: ts.PromMetricTypeCounter, handleValueResets: true},
+		{prompb.MetricType_COUNTER, "total"}:    {metricType: ts.PromMetricTypeCounter, handleValueResets: true},
+		{prompb.MetricType_COUNTER, "created"}:  {metricType: ts.PromMetricTypeCounter},
+
 		{metricType: prompb.MetricType_GAUGE}:    {metricType: ts.PromMetricTypeGauge},
 		{metricType: prompb.MetricType_INFO}:     {metricType: ts.PromMetricTypeInfo},
 		{metricType: prompb.MetricType_STATESET}: {metricType: ts.PromMetricTypeStateSet},
 
-		{prompb.MetricType_HISTOGRAM, "bucket"}: {metricType: ts.PromMetricTypeHistogram, handleValueResets: true},
-		{prompb.MetricType_HISTOGRAM, "count"}:  {metricType: ts.PromMetricTypeHistogram, handleValueResets: true},
-		{prompb.MetricType_HISTOGRAM, "sum"}:    {metricType: ts.PromMetricTypeHistogram, handleValueResets: true},
+		{prompb.MetricType_HISTOGRAM, "bucket"}:  {metricType: ts.PromMetricTypeHistogram, handleValueResets: true},
+		{prompb.MetricType_HISTOGRAM, "count"}:   {metricType: ts.PromMetricTypeHistogram, handleValueResets: true},
+		{prompb.MetricType_HISTOGRAM, "sum"}:     {metricType: ts.PromMetricTypeHistogram, handleValueResets: true},
+		{prompb.MetricType_HISTOGRAM, "created"}: {metricType: ts.PromMetricTypeHistogram},
 
-		{prompb.MetricType_GAUGE_HISTOGRAM, "bucket"}: {metricType: ts.PromMetricTypeGaugeHistogram},
-		{prompb.MetricType_GAUGE_HISTOGRAM, "count"}:  {metricType: ts.PromMetricTypeGaugeHistogram, handleValueResets: true},
-		{prompb.MetricType_GAUGE_HISTOGRAM, "sum"}:    {metricType: ts.PromMetricTypeGaugeHistogram},
+		{prompb.MetricType_GAUGE_HISTOGRAM, "bucket"}:  {metricType: ts.PromMetricTypeGaugeHistogram},
+		{prompb.MetricType_GAUGE_HISTOGRAM, "count"}:   {metricType: ts.PromMetricTypeGaugeHistogram, handleValueResets: true},
+		{prompb.MetricType_GAUGE_HISTOGRAM, "sum"}:     {metricType: ts.PromMetricTypeGaugeHistogram},
```

Comment:
I believe this might be a mistake. [OpenMetrics spec mentions](https://github.com/OpenObservability/OpenMetrics/blob/45ae6699a4ba83368f0b481f05e798ba3c398eec/specification/OpenMetrics.md#suffixes) `_gcount` and `_gsum` suffixes.